assets/js/components/flows/FlowsWorkspace.jsx

```diff
@@ -236,6 +240,28 @@ export default ({
         ref={reactFlowWrapper}
         style={{ position: "relative", height: "100%", width: "100%" }}
       >
+        {
+          (organization.flow === "{\"edges\":[],\"copies\":[]}" || organization.flow === "{}") && (
+            <div
+              style={{
+                position: "absolute",
+                bottom: 20,
+                right: 100,
+                zIndex: 100,
+                paddingLeft: 20,
+              }}
+            >
+              <div className="pod" id="left">
+                <RocketFilled className="bigicon" />
+                <h2>Get Started with Console</h2>
+                <Link to="/devices"><p>Add and Manage Devices <span>for the Helium Network</span><CaretRightOutlined className="caret" /></p></Link>
+                <Link to="/integrations"><p>Set up an Integration <span>to send and receive device data</span><CaretRightOutlined className="caret" /></p></Link>
+                <Link to="/functions"><p>Apply Functions <span>to your devices</span><CaretRightOutlined className="caret" /></p></Link>
+                <Link to="/users"><p>Invite other Users <span> to your Console Organization</span><CaretRightOutlined className="caret" /></p></Link>
+              </div>
```

Comment:
could we extract this from `Welcome.jsx` as well from here into its own component and reuse? would be easier if we ever have to update links or anything here